msg/en_US/utils.msg

```diff
@@ -973,6 +973,7 @@ valid options:\n\
 $set 44 MSGCAT_UTIL_SET_APPLYINFO
 21 Enter DBA password:
 22 The server was not configured for HA.\n
+57 Can't generate the applied info due to an invalid log volume path or a problem in db_ha_apply_info catalog table. \n
```

Comment:
How about the following message?

Can't generate the applied info due to an invalid path of the -L option or no related information in the db_ha_apply_info catalog table.

Reply:
After the error message is confirmed, please add this to other utils.msg files for multilingual support.

Reply:
sure. Thank you.